enterprise/enrollment_api.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to get details from the course catalog API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import requests
+from edx_rest_api_client.client import EdxRestApiClient
+
+from django.conf import settings
+
+
+def get_enrollment_api_client():
+    """
+    Retrieve a client of the Enrollment API.
+
+    The client is authenticated using the API token from the Django settings.
+    """
+    session = requests.Session()
+    session.headers = {"X-Edx-Api-Key": settings.EDX_API_KEY}
+    return EdxRestApiClient(
+        settings.ENTERPRISE_ENROLLMENT_API_URL, append_slash=False, session=session
```

Comment:
what will happened if `ENTERPRISE_ENROLLMENT_API_URL` is not defined in settings ? Should we not retrieve it in safe mode e.g. `getattr` ?